lms/djangoapps/verify_student/tasks.py

```diff
@@ -10,9 +10,16 @@
 from celery import Task, task
 from celery.states import FAILURE
 from django.conf import settings
+from django.contrib.auth.models import User
+from django.contrib.sites.models import Site
 from django.core.mail import EmailMessage
+from edx_ace import ace
+from edx_ace.recipient import Recipient
 
+from openedx.core.djangoapps.ace_common.template_context import get_base_template_context
 from edxmako.shortcuts import render_to_string
+from lms.djangoapps.verify_student.message_types import VerificationApproved
+from openedx.core.djangoapps.lang_pref import LANGUAGE_KEY
 from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
```

Comment:
Where are these imports used?

Reply:
These are ordered using `isort` to remove the quality errors.

Reply:
They have been added, not re-ordered.

Reply:
Taking a look.!

Reply:
Updated.